### PR TITLE
modals instead of alerts for battle outcomes and login failures and s…

### PR DESCRIPTION
**Pull request by bowenbrinegar**
…uccesses

### CODE REVIEW

**Comment by bowenbrinegar** [comment]
Agreed. Fixed.

File: assets/javascript/config.js [hunk 1]
```diff
@@ -81,54 +81,21 @@ $('#closeBattle').on("click", function() {
...
   infoModal.style.display = 'block';
 });
 
-$(infoClose).on('click', function(){
+$('#modalBox').on("click", "span", function(){
   infoModal.style.display = 'none';
-});
-
-$(failureClose).on('click', function(){
   loginFailure.style.display = 'none';
```
Comment (kcgilland143) [inline assets/javascript/config.js]: are these variables still defined? EDIT: nevermind that part. also, you could do a document click handler on '.closer' and do your style.display on this.parentElement (or something like that, sorry, away from computer)

**Review by kcgilland143 — COMMENTED** [review]
finally was able to check it out in game! they look nice and I'm liking where it's going! It just doesn't feel like you're done with it. if that's the case, i'm not sure we should merge it yet. at the same time, i'm not sure it would hurt anything either. maybe we should leave it open at least so Jane can see the changes

**Comment by bowenbrinegar** [comment]
Yeah we can leave this open for @janeaguero

**Pull request closed without merging**